x-pack/legacy/plugins/apm/public/components/app/ErrorGroupDetails/Distribution/index.tsx

```diff
@@ -61,7 +63,7 @@ export function ErrorDistribution({ distribution, title }: Props) {
     distribution.bucketSize
   );
 
-  if (distribution.noHits) {
+  if (!buckets || distribution.noHits) {
```

Comment:
why do we need the additional `buckets` check? what changed?

Reply:
I added only for safety reason, as buckets can be `FormattedBucket[] | null`